### PR TITLE
Update new-member-welcome.md

### PR DESCRIPTION
**Pull request by husseinalhammad**
Updating the new members document with updated information.

### CODE REVIEW

File: new-member-welcome.md [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-
-Once we have things more set up we are planning on cutting the weekly call back to a fortnightly or monthly call. We are also planning on a public forum to have more open discussion and allow more members of the community to share ideas. And a public website to publish our documentation.
+### Forum
+We aim to keep our discussions public whenever possible. This typically happens in the [Discussions](https://github.com/email-markup-consortium/email-markup-consortium/discussions) section of this repository which acts as our public forum.
 
-[Notes from previous calls can be found here](https://github.com/email-standards/email-standards/tree/main/meeting-notes). Also there is an agenda at the top of the page for the next call. Please feel free to add items for discussion.
+### Chat
+We also chat on Slack in the [Email Geeks Slack community](https://email.geeks.chat/) in the #email-markup-consortium channel. If you are an approved member and still do not have access to this channel, please contact [EMC admins](admins@emailmarkup.org).
```
Comment (M-J-Robbins) [inline new-member-welcome.md:10]: I think this needs to be a `mailto:` link

**Review by M-J-Robbins — CHANGES_REQUESTED** [review]
One small change but otherwise all good to go

File: new-member-welcome.md (hunk 1 above)
Comment (husseinalhammad) [reply new-member-welcome.md]: Updated

**Review by M-J-Robbins — APPROVED** [review]
All good 👍

**Pull request merged**